lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -32,6 +36,9 @@ class UserProfileBloc {
   final _currencyController = new BehaviorSubject<Currency>();
```

Comment:
We have here two controllers:
1. _currencyController - not used should be deleted.
2. _fiatConversionController - which is used to detect currency changes, I think better to call it _currencyController as only accountBloc is dealing with conversion.